src/main/java/org/synyx/urlaubsverwaltung/config/SwaggerConfig.java

```diff
@@ -1,52 +1,48 @@
 package org.synyx.urlaubsverwaltung.config;
 
-import com.mangofactory.swagger.configuration.SpringSwaggerConfig;
-import com.mangofactory.swagger.models.dto.ApiInfo;
-import com.mangofactory.swagger.plugin.EnableSwagger;
-import com.mangofactory.swagger.plugin.SwaggerSpringMvcPlugin;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.service.Contact;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import java.util.Collections;
 
 
 /**
  * @author  Aljona Murygina - murygina@synyx.de
  */
 @Configuration
-@EnableSwagger
+@EnableSwagger2
 public class SwaggerConfig {
```

Comment:
würde hier noch eine Option anbieten, swagger auszuschalten - je nachdem, ob man das in Produktion verwenden will oder auch nicht